ci-operator/config/insights-qe/iqe-cost-management-plugin/insights-qe-iqe-cost-management-plugin-master__ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: cost-management-interop-aws
+  cron: 0 6 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+         {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+         {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+         {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
+         {"step": "insights-qe-iqe-*", "failure_type": "test_failure", "classification": "Cost Management Test Failure", "jira_project": "RHIROS", "jira_epic": "LPTOCPCI-290"}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+    test:
+    - ref: insights-qe-iqe-cost-management-plugin
+    workflow: firewatch-ipi-aws
+  timeout: 8h0m0s
```

Comment:
Why this long timeout?

Reply:
@Simranpal PQE plans to add more tests after the initial trial is successful. We don't want to give out a timeout error unless there is any specific requirement. 8 hours is a valid and max number to set.